homework-submission/week3-homework/src/components/AddTodoForm.js

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+class AddTodoForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.onAddTodo = this.onAddTodo.bind(this);
+  }
+
+  onAddTodo(e) {
+    e.preventDefault();
+    let newTodo = this.enteredTodo.value;
+    if (newTodo === "") {
+      return "";
+    }
+    this.props.AddTodo(newTodo);
+    this.enteredTodo.focus();
```

Comment:
I wanted to make sure the focus is on input field even when you click on button.Because when I enter todo by clicking on `Add`  button the focus is staying on button, by writing this line `this.enteredTodo.focus();` focus stays on input field which is more user firendly so they would not have to clicking back on input field everytime wanting to add a todo.